### DISCUSSION TITLE
[Feature] [remix adapter]: Allow user to specify host port

### DISCUSSION THREAD

**Discussion by dima-takoy-zz** [remix-run/remix#2908] [Proposals]
### What is the new or updated feature that you are suggesting?

When i develop i frequently check website on real ios/android device. 

### Why should this feature be included?

When i develop i frequently check website on real ios/android device, also its an easy way for sharing local development version with collegues.

P.S:
Next.js runs on 0.0.0.0 by default, but i personally think that it is unsafe by default.

**Comment by sergiodxa** [comment]
Since in Remix you are in control of the server you could configure that yourself

If you are using the Express adapter you can configure it in the Express server code.

If you are using the Remix App Server, the host is not configurable which would be a nice feature to add, but you should be able to switch to the Express adapter quite easy since the Remix App Server uses that internally.
